lib/ansible/modules/cloud/amazon/dynamodb_table.py

```diff
@@ -459,6 +470,12 @@ def main():
     if not region:
         module.fail_json(msg='region must be specified')
 
+    if ec2_url:
```

Comment:
I wonder if this should be done in module_utils.ec2.boto3_conn rather than here - it seems like it should be reusable elsewhere (I really don't know how common local implementations of AWS services are, but guess dynamodb isn't the only one - it might (or might not) help with some of the S3 URL problems we've had recently)

Reply:
I'd be happy to rewrite this module to support only boto3, but that would break backward compatibility since this module requires `boto3` only to tag tables and uses older version of `boto` for everything else. 
That's why it's right here, in the module. As far as I can see there's no wrapper  in ansible around old `boto` that abstracts connection to AWS.

Reply:
It's not super common for AWS to support local implementations - S3 is by far the most used since many vendors implement the S3 API themselves. 

As far as boto3, it would actually be fine to refactor this module to boto3 only now that boto3 is so commonly available, and users on modern ansible installs are likely to be using at least one boto3-only module already.